### DISCUSSION TITLE
Files over 10mb wont play or download on RHEL8.

### DISCUSSION THREAD

**Discussion by 01beacon** [requarks/wiki#5881] [Help / Questions]
Hi, we have a problem. In our corporation we have set up a new wiki. We went with wiki.js on a **Red hat 8 Enterprise** server and **ProxyPass** with **Apache**. The problem we have is that when uploading files bigger than 10mb, we can't open or download them, the server give us an error. **HTTP Error message 500 21**, (Internal Server Error). All the files under 10mb works just fine. It can be a pdf, a mp4 or some other file, it doesn't matter. 

The upload limit is changed and the uploading process of files bigger than 10mb works fine, (at least it looks like that at the ui side)... ....in the background we get a s*it loads of error message in rsyslog that "message too long (49152) with configured size 8096, begin of message is: f1ed51c87dc58bfd8ef8f5......................" when uploading. Can this be related to the problem?

It's worth to say that we have tried with SELinux disabled, same problem. We have tried to access the wiki directly on port 3000 and make the upload, same problem. Now we are stuck, can't figure out what it can be and the logs doesn't give enough info for us to get wiser. 

Would really appreciate all help we can get and if you need more info, please just say so. 

**Thanks!**

**Comment by cdn0x12** [comment]
Have you tried increasing `$MaxMessageSize` in `/etc/rsyslog.conf`? (If you route your logs to separate files then a specific child config should be changed). The problem might be relevant to the system rather than wiki.js

**Comment by 01beacon** [comment]
@CDN18 Thanks for fast response. Yes we have tried that, even disabled some logs and now we doesn't get that error anymore. 

When uploading the "big_file_test.mp4" for example, this is what happen in our logs. (Same with PDF, Word or anything over 10mb)
```
==> /etc/httpd/logs/ssl_request_log <==
[23/Nov/2022:16:03:00 +0100] user_ip TLSv1.3 TLS_AES_256_GCM_SHA384 "POST /u HTTP/1.1" 2
[23/Nov/2022:16:03:00 +0100] user_ip TLSv1.3 TLS_AES_256_GCM_SHA384 "POST /graphql HTTP/1.1" 16905
```
**And when saving and viewing the page.**
```
==> /var/log/messages <==
Nov 23 16:03:02 wikiserver node[99034]: Loading configuration from /var/www/wiki/config.yml... OK
```
```
==> /etc/httpd/logs/ssl_request_log <==
[23/Nov/2022:16:03:02 +0100] user_ip TLSv1.3 TLS_AES_256_GCM_SHA384 "POST /graphql HTTP/1.1" 285
[23/Nov/2022:16:03:22 +0100] user_ip TLSv1.3 TLS_AES_256_GCM_SHA384 "GET /logo_64x64.png HTTP/1.1" -
[23/Nov/2022:16:03:23 +0100] user_ip TLSv1.3 TLS_AES_256_GCM_SHA384 "POST /graphql HTTP/1.1" 642
[23/Nov/2022:16:03:22 +0100] user_ip TLSv1.3 TLS_AES_256_GCM_SHA384 "GET /small_file.mp4 HTTP/1.1" 327608
[23/Nov/2022:16:03:57 +0100] user_ip TLSv1.3 TLS_AES_256_GCM_SHA384 "GET /big_file_test.mp4 HTTP/1.1" 21
```
```
==> /var/log/messages <==
Nov 23 16:03:57 wikiserver node[94485]: 2022-11-23T15:03:57.956Z [MASTER] #033[31merror#033[39m: Cannot read property 'data' of undefined
```
